tests/providers/elasticsearch/log/test_es_task_handler.py

```diff
@@ -251,6 +251,31 @@ def test_set_context_w_json_format_and_write_stdout(self):
         self.es_task_handler.json_format = True
         self.es_task_handler.set_context(self.ti)
 
+    def test_read_with_json_format(self):
+        ts = pendulum.now()
+        formatter = logging.Formatter('[%(asctime)s] {%(filename)s:%(lineno)d} %(levelname)s - %(message)s')
+        self.es_task_handler.formatter = formatter
+        self.es_task_handler.json_format = True
```

Comment:
maybe invalid question: `self.es_task_handler` is also used in other test cases, and default/initial value of `json_format` is `False` here. So should it be changed back to `False` in the end (or in `tearDown`) to avoid potential effect on other test cases (especially the order of test case execution is not guaranteed if I'm not wrong)?

the same quesiton applys to `test_set_context_w_json_format_and_write_stdout ` above.

Reply:
I don't think that is needed since we already set it back to False in `setUp` before any tests using `self.es_task_handler` is run : 

https://github.com/apache/airflow/blob/69d6d0239f470ac75e23160bac63408350c1835a/tests/providers/elasticsearch/log/test_es_task_handler.py#L48-L55

Reply:
Yep, you are right. I asked a silly question